care/facility/migrations/0424_alter_fileupload_file_category_and_more.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 4.2.10 on 2024-04-02 11:49
+
+from django.db import migrations, models
```

Comment:
Why is this migration created even thou we had no model change?

Reply:
The case was changed, that caused the migration, can you check if we are okay with the new case, there are no reverse lookups.